typedoc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "BpmnVisualization API Documentation",
+  "readme": "none",
+  "out": "./build/docs/api",
+  "excludeInternal": true,
+  "excludeExternals": true,
+  "excludePrivate": true,
+  "excludeProtected": true
```

Comment:
❓ shouldn't we keep protected API for extension?
Or do we activate it when we work on extension support?

Reply:
I have excluded it as we do not work on the extension topic yet.
But let's remove it, not really crucial at the moment.